src/system/tests/opposed-result.js

```diff
@@ -20,6 +20,54 @@ export class OpposedTestResult
         {
             this.winner = "defender";
         }
+        else {
+            // If both parties in an Opposed Test get the same SL, the character with the higher Skill wins
+            if (defenderTest)
+            {
+                const getSkillSpecTotal = test =>
+                {
+                    let skillObject;
+                    if (typeof test.skill == "string")
+                    {
+                        skillObject = test.actor.system.skills[test.skill];
+                    }
+                    else if (test.skill instanceof Item)
+                    {
+                        skillObject = test.skill.system;
+                    }
+                    else if (typeof test.skill == "object")
+                    {
+                        skillObject = test.skill;
+                    }
+                    return skillObject.getTotalFor(skillObject.characteristic, test.actor);
+                };
```

Comment:
I think I would recommend adding a getter to each test class to get the appropriate comparison number instead of a bunch of if/else statements, so it would instead compare something like `attackerTest.tiebreakerValue` and `defenderTest.tiebreakerValue`

Reply:
Just realised I can use `SkillTest.computeTarget(base=true)` here instead to get the total skill/spec value.